tests/test_schema.py

```diff
@@ -186,12 +184,27 @@ def test_raises_usetypeinvalid(type_):
         assert schema.UseType(type_) == type_
 
 
+def test_takedown():
+    """Test that a valid takedown passes validation."""
+    full_doc = load_json('valid.json')
+    full_doc['action'] = 'takedown'
+    actual_full_doc = schema.validate_schema(schema.takedown, full_doc)
+    assert actual_full_doc == full_doc
```

Comment:
This is a common pattern we use throughout our tests:

    assert actual == expected

You should rename 'full_doc` to `expected` and `actual_full_doc` to `actual`.